addons/im_livechat/views/im_livechat_chatbot_templates.xml

```diff
@@ -17,7 +17,7 @@
                     <div groups="im_livechat.im_livechat_group_user" t-ignore="true"
                             class="alert alert-info alert-dismissible rounded-0 fade show d-print-none css_editable_mode_hidden mb-0">
                         <div t-ignore="true" class="text-center">
-                            <a t-attf-href="/web#view_type=form&amp;model=chatbot.script&amp;id=#{chatbot_data['id']}&amp;action=im_livechat.chatbot_script_action">
+                            <a t-attf-href="/odoo/action-im_livechat.chatbot_script_action/{chatbot_data['id']}">
```

Comment:
```suggestion
                            <a t-attf-href="/odoo/action-im_livechat.chatbot_script_action/#{chatbot_data['id']}">
```

Reply:
Hello @jpp-odoo 
`<a t-attf-href="/odoo/action-im_livechat.chatbot_script_action/#{chatbot_data['id']}">`
Implied above changes, Please check it.
Thanks!!